core-bundle/src/Twig/Extension/ContaoCompatExtension.php

```diff
@@ -0,0 +1,32 @@
+<?php
+
+declare(strict_types=1);
+
+/*
+ * This file is part of Contao.
+ *
+ * (c) Leo Feyer
+ *
+ * @license LGPL-3.0-or-later
+ */
+
+namespace Contao\CoreBundle\Twig\Extension;
+
+use Contao\CoreBundle\Twig\Compat\SafeHTMLValueHolderInterface;
+use Twig\Environment;
+use Twig\Extension\AbstractExtension;
+use Twig\Extension\EscaperExtension;
+
+class ContaoCompatExtension extends AbstractExtension
+{
+    /**
+     * @internal
```

Comment:
Why not at class level?

Reply:
Your IDE at least marks it when you create an object 🙃